meta/dev.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: transactors
+  roles:
+    - { role: jpnewman.java, tags: ["init"], become: yes, become_user: "root" }
+    - { role: milinda.sbt, become: yes, become_user: "root" }
+    - { role: dynamodb_local, become: yes, become_user: "root" }
```

Comment:
you reall want to run those as root? use a less priviledged user please, even if it's just for local testing.